subsys/net/l2/openthread/Kconfig

```diff
@@ -150,6 +150,24 @@ config OPENTHREAD_JOINER
 	help
 	  Enable joiner capability in OpenThread stack
 
+config OPENTHREAD_JOINER_AUTOSTART
+	bool "Start joiner automatically support"
+	depends on OPENTHREAD_JOINER
+	help
+	  Enable starting joiner automatically
+
+config OPENTHREAD_JOINER_PSKD
+	string "Default Pre Shared key for the Device to start joiner"
+	default "J01NME"
+	help
+	  Pre Shared Key for the Device to start joiner
+
+config OPENTHREAD_PLATFORM_INFO
+	string "Platform infomation of OpenThread"
```

Comment:
Just noticed, please make this `Platform information for OpenThread` here and below